src/impl/tpls/KokkosBlas_Lapack_tpl.hpp

```diff
@@ -0,0 +1,43 @@
+#ifndef KOKKOSBLASLAPACK_LAPACK_TPL_HPP_
+#define KOKKOSBLASLAPACK_LAPACK_TPL_HPP_
+
+#include<KokkosBlas_Host_tpl.hpp> // trtri prototype
+#include<KokkosBlas_tpl_spec.hpp> // LAPACKE_
+
+#if defined(KOKKOSKERNELS_ENABLE_TPL_LAPACK)
+
+namespace KokkosBlas {
+  namespace Impl {
+    //// TRTRI ////
+    template<>
+    int 
+    HostBlas<double >::trtri(const int matrix_layout, const char uplo, const char diag,
+                                         int n, // TODO: n,lda lapack_int
+                                         const double *a, int lda) {
+      return LAPACKE_dtrtri(matrix_layout, uplo, diag, n, const_cast <double*> (a), lda);
```

Comment:
Why would we need LAPACKE interfaces? Can we just use Fortran interfaces instead?

Reply:
Same as above. There are no fortran interfaces for `trtri`.

Reply:
@e10harvey I think there are. Please check this out https://software.intel.com/en-us/mkl-developer-reference-fortran-trtri. I used Fortran interfaces for ```gesv```